db/schema.rb

```diff
@@ -140,6 +140,7 @@
     t.string "encrypted_source_data"
     t.string "encrypted_source_data_iv"
     t.string "header_md5"
+    t.string "signature_errors", default: [], array: true
```

Comment:
Are we expecting to get any errors back from BGS at any point? Should we just call this errors for now and we can combine any signature based ones vs any errors from BGS/VBMS into one error list?

Reply:
Possibly, but these errors were to be specific about bad signature b64 data since json schema doesnt really give us alot of options. maybe we need 3 columns. SignatureErrors, VBMS error, BGS error, and then the serializer can combine them into one error field